test/models/mission_test.rb

```diff
@@ -4,4 +4,35 @@ class MissionTest < ActiveSupport::TestCase
   # test "the truth" do
   #   assert true
   # end
+  def test_the_truth
```

Comment:
Please always name your tests so that it is clear what they are there for. The method `test_the_truth` is an example that - well - tests the truth and will always pass. Naming your test like that is an indicator that you are very new to testing. This isn't bad. Everyone has to start at some point. But you should give meaningful names from now on.